sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -141,17 +143,42 @@ internal class ViewAnnotationManagerImpl(
   private fun prepareViewAnnotation(inflatedView: View, options: ViewAnnotationOptions): View {
     checkAssociatedFeatureIdUniqueness(options)
     val inflatedViewLayout = inflatedView.layoutParams as FrameLayout.LayoutParams
+    // provide explicit 1 px sizes if inflated sizes are zero (use-case for wrap_content, match_parent)
```

Comment:
Why 1 is needed?

Reply:
cause width or height = 0 are not supported as dimensions in core part. I guess we could potentially change overall logic to not set sizes to 1 but do not call `core.addViewAnnotation` at all until dimensions are calculated, wdyt?

Reply:
Looking again at overall logic - it will be a hard change so indeed it's easier to set firstly width / height to non-null dimensions so that core will process it, trigger redraw, we will add the view to root layout and update with correct dimensions as soon as view is measured
Let me know if you have other thoughts here

Reply:
Sounds good, maybe just add this explanation as a comment

Reply:
Added more comments